tests/sprinkles/sprinkles.test.ts

```diff
@@ -2,7 +2,7 @@ import {
   createMapValueFn,
   createNormalizeValueFn,
 } from '@vanilla-extract/sprinkles';
-import { createAtomsFn } from '@vanilla-extract/sprinkles/createRuntimeAtomsFn';
+import { createAtomsFn } from '@vanilla-extract/sprinkles';
```

Comment:
This shows the fix working.

Reply:
This should be merged into the import statement above it.